amundsen_application/static/js/components/Feedback/styles.scss

```diff
@@ -5,7 +5,7 @@
   bottom: 75px;
   position: fixed;
   right: 25px;
-  z-index: 2;
+  z-index: 999;
```

Comment:
Ah, yeah there was an issue with the overlapping user avatars (had to modify z-indexes to get the avatars to render in the correct order). Should I just make this value smaller?

Reply:
yeah the bubbles would just go over the component on a small page, figured I'd put in a quick fix in this PR.

Reply:
I see, good eye catching the bug! Though as it stands it will make more sense to fix this either on master or as part of the update to put this in the NavBar.